inference/pyproject.toml

```diff
@@ -1,14 +1,12 @@
 [tool.poetry]
-name = "openpipe-inference"
+name = "inference"
 version = "0.1.0"
 description = ""
 authors = ["Kyle Corbitt <kyle@corbt.com>"]
 readme = "README.md"
 
 [tool.poetry.dependencies]
-python = "^3.9"
-anyscale = "^0.5.171"
-ray = {extras = ["serve"], version = "^2.9.0"}
+python = "^3.12"
```

Comment:
Why are we specifying version 3.9 in .tool-versions and 3.12 in pyproject.toml?

Reply:
Good catch; needs to be 3.9 because Ray doesn't have a 3.12 dist available. Updated to make this consistent.